nodes/image_processing.py

```diff
@@ -1,5 +1,6 @@
 import comfy  # type: ignore
 import folder_paths  # type: ignore
+import math
```

Comment:
This seems not to be sorted with isort. Ruff should apply isort, we have the setting in pyproject.toml. Maybe we can check what is different in your setup?